trikot-viewmodels/viewmodels/src/commonMain/kotlin/com/mirego/trikot/viewmodels/ViewModel.kt

```diff
@@ -23,4 +23,18 @@ interface ViewModel {
      * Action to execute when the view is tapped
      */
     val action: Publisher<ViewModelAction>
+    /**
+     * Return true if it should be exposed as an accessibility element.
+     * Note : `accessibilityLabel` or `accessibilityHint` must be specified (at least one) if this property is set to true or it will have no impact.
+     */
+    val isAccessibilityElement: Publisher<Boolean>
+    /**
+     * String that identifies the accessibility element
+     */
+    val accessibilityLabel: Publisher<String>
+    /**
+     * Describes the result of performing an action on the element, when the result is non-obvious.
+     * For example: "Purchases the item." or "Downloads the attachment."
+     */
+    val accessibilityHint: Publisher<String>
```

Comment:
These should all be part of another interface, not the on ViewModel itself. Let's say :

```kotlin
interface Accessible {
    val accessibilityLabel: Publisher<String>
    val accessibilityHint: Publisher<String>
```

Reply:
Also, I feel like these should not be publishers, but instead static elements. It seems weird for the user to have inconsistent values when running through accessibility elements. What do you think @dominiclabbe ?

Reply:
All those values can change frequently. For example, the same button that is used to play and pause a video will have a different `accessibilityHint` each time we interact with it in the same way we alternate the resources between the "pause" and "pause" image.

I agree however that we could extract the new accessibilities properties in an `Accessible` interface like you propose. 👍 Or maybe something like `ViewModelAccessible` to limit the potential name conflicts?

Reply:
You're right, it actually makes sense with reactive components. 👍🏻 It might be different with declarative UI, though 👀

Reply:
That's what we did in Fibe, every component we want to make accessible implements the `Accessible` interface.